frontend/src/views/DashboardView/components/QuickActionsSideBar.vue

```diff
@@ -0,0 +1,122 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { useUserStore } from '@/stores/user-store';
+
+const { t } = useI18n();
+const user = useUserStore();
+
+const props = defineProps<{
+  pendingBookingRequestsCount: number
+}>()
```

Comment:
I think we explicitly set `;` (maybe we should really run the frontend linter at some point 😂 )

Reply:
Hm, if `npm run lint` doesn't do it, then I think I was imagining things 😂  I have the same formatter like you. So guess it was another project then. Personally I like to set semicolons for better readable code. I'll take a look at the eslint config and see if that is configured correctly. I'm not so much of a pre-commit-hook-fan myself, in my experience this breaks the dev flow more than it helps. But good point anyway!